taskotron_python_versions/requires.py

```diff
@@ -0,0 +1,62 @@
+import collections
+
+from .common import log, write_to_artifact
+from .naming_scheme import is_unversioned
+
+MESSAGE = """These RPMs use `python-` prefix without Python version in Requires:
+{}
+This is strongly discouraged and should be avoided. Please check
+the required packages, and use names with either `python2-` or
+`python3-` prefix if available.
+"""
+
+# Should be a link to guidelines when
+# https://pagure.io/packaging-committee/issue/686 accepted.
+INFO_URL = ''
```

Comment:
This should probably be a link to guidelines, when https://pagure.io/packaging-committee/issue/686 is considered.